### ISSUE TITLE
Can someone publish new version of ideabuck plugin to JetBrains repository

### ISSUE THREAD

**Issue by styurin** [facebook/buck#2361]
The last version is from Sep 18, 2018.

**Comment by styurin** [comment]
@asp2insp can someone help with that?

**Comment by philipjameson** [comment]
Hey sergey, I think someone has sent it off to jetbrains for approval. I've asked them to follow up this coming week on the status of that.

**Comment by philipjameson** [comment]
Still waiting on jetbrains :/

**Comment by styurin** [comment]
Thanks for the update.

**Comment by styurin** [comment]
It's still showing as pending approval. Is it stuck somewhere in their process?

**Comment by philipjameson** [comment]
Yeah, it's seeming that way. Someone's going to dig in later today to see what needs to be done to unstick things.
